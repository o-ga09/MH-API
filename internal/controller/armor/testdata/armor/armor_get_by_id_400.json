{
  "message": "Invalid request parameters: Key: 'GetArmorByIDRequest.ArmorID' Error:Tag: 'required' Error Field: 'ArmorID' Error Value: '' Error: Error:Field validation for 'ArmorID' failed on the 'required' tag"
}