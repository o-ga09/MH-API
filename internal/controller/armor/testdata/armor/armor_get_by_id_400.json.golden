{
  "message": "Armor ID cannot be empty"
}